### PR TITLE
Expose test duration

### LINKED ISSUE #282: Expose per-test execution time
As https://github.com/web-platform-tests/wpt/issues/10026 the wptreport JSON has the execution time at a per-test level. Exposing this in the UI would be great, and even greater would be the ability to search for slow tests.

### PR DESCRIPTION
**Pull request by lukebjerring**
## Description
Fixes #282 
Surfaces the `duration` field of each single test-result-file.

## Review Information
- Visit the deployed environment
- View any individual test
- See "Duration" row underneath the main status row.

### CODE REVIEW

File: webapp/components/test-file-results-table.js [hunk 1]
```diff
@@ -133,22 +133,22 @@ class TestFileResultsTable extends WPTColors(PathInfo(TestRunsBase)) {
...
-                <template is="dom-if" if="[[ testScreenshot(result.screenshots) ]]">
-                  <a href="[[ computeAnalyzerURL(result.screenshots) ]]">
-                    <img src="[[ testScreenshot(result.screenshots) ]]" />
-                  </a>
-                </template>
-              </td>
-            </template>
-          </tr>
-        </template>
+    <template is="dom-if" if="[[verbose]]">
```
Comment (lukebjerring) [inline webapp/components/test-file-results-table.js]: This is simply moving the whole dom-if out of the `<template is="dom-repeat">` - we were producing an extra dom-if per row, when this dom-if only applies to screenshot (which have just one row). That meant that the css `nth-child(2)` selector was hitting the dom-if, not the 2nd `tr`.

File: webapp/components/test-file-results.js [hunk 2]
```diff
@@ -164,18 +165,26 @@ class TestFileResults extends WPTFlags(LoadingState(PathInfo(
...
+            status: data && data.status,
+            message: data && data.message,
+          };
+          if (this.reftestAnalyzer && data && data.screenshots) {
+            result.screenshots = this.shuffleScreenshots(this.path, data.screenshots);
+          }
+          return result;
+        })
+      },
+      {
```
Comment (lukebjerring) [inline webapp/components/test-file-results.js]: This is the interesting change. Adds a "results row" for duration.

**Comment by foolip** [comment]
@Hexcles can you review? It's not deployed yet and I should run :)

**Comment by lukebjerring** [comment]
https://github.com/web-platform-tests/wpt.fyi/pull/1528 fixed the deployment.

**Pull request merged**